docker/docker_build.sh

```diff
@@ -16,6 +16,10 @@
 #   This make the Docker image reproducible across different machines.
 set -euo pipefail
 
+# Disable Docker build kit to show all outputs, as `--progress plain`` does not
+# work on all systems.
+export DOCKER_BUILDKIT=0
```

Comment:
Seems like it only works for "parallel multistage builds". Since our builds are not *multistage*, it might not be much faster. 

When run locally, the `docker_build.sh` is typically used for testing or recreating CI issues locally. Since it is debugging only, it is good to have more verbose output.